packages/web_benchmarks/testing/test_app/lib/benchmarks/runner.dart

```diff
@@ -97,3 +97,11 @@ Future<void> main() async {
     'tap': () => TapRecorder(),
   });
 }
+
+/// This allows a value of type T or T? to be treated as a value of type T?.
+///
+/// We use this so that APIs that have become non-nullable can still be used
+/// with `!` and `?` on the stable branch.
+// Todo(amanv8060): Remove this once https://github.com/flutter/flutter/pull/89451
+// lands to stable . See https://github.com/flutter/flutter/issues/64830
+T? _ambiguate<T>(T? value) => value;
```

Comment:
Do we need pre-Flutter-3.0 support in this package? If not, you can remove this and just change the Flutter SDK constraint.